src/profile/profile.spec.ts

```diff
@@ -73,17 +73,43 @@ describe('Profile helper', function test() {
     expect(await profile.getContactKey(identities[2], 'context a')).to.be.undefined;
   });
 
-  it('should be able to be add dapp bookmarks', async () => {
+  it('should be able to be add identity key', async () => {
+    await profile.loadForAccount(profile.treeLabels.addressBook);
+    await profile.setIdentityAccess(identities[0], 'key 0x01_a');
+    const list = await profile.getIdentityAccessList();
+    expect(list).to.be.not.empty;
+
+    await profile.storeForAccount(profile.treeLabels.addressBook);
+    await profile.loadForAccount(profile.treeLabels.addressBook);
+    const loadedList = await profile.getIdentityAccessList();
+    expect(loadedList).to.be.deep.eq(list);
+  });
+
+  it('should remove an identity key', async () => {
+    await profile.loadForAccount(profile.treeLabels.addressBook);
+    const beforeSettingList = await profile.getIdentityAccessList();
+
+    await profile.setIdentityAccess(identities[1], 'key 0x01_b');
+    await profile.storeForAccount(profile.treeLabels.addressBook);
+    await profile.loadForAccount(profile.treeLabels.addressBook);
+
+    await profile.removeIdentityAccess(identities[1]);
+    await profile.storeForAccount(profile.treeLabels.addressBook);
+    await profile.loadForAccount(profile.treeLabels.addressBook);
+    const afterRemovingList = await profile.getIdentityAccessList();
+    expect(afterRemovingList).to.be.deep.eq(beforeSettingList);
```

Comment:
What happens, when the key was already added to the profile before this test is running? Could we change the check to a `include` test?

(Code is not tested, please check again, when you include it.)

```suggestion
    await profile.loadForAccount(profile.treeLabels.addressBook);
    await profile.setIdentityAccess(identities[1], 'key 0x01_b');
    await profile.storeForAccount(profile.treeLabels.addressBook);
    await profile.loadForAccount(profile.treeLabels.addressBook);
    
    const beforeSettingList = await profile.getIdentityAccessList();
    expect(beforeSettingList[identities[1]]).to.be.eq('key 0x01_b');

    await profile.removeIdentityAccess(identities[1]);
    await profile.storeForAccount(profile.treeLabels.addressBook);
    await profile.loadForAccount(profile.treeLabels.addressBook);
    const afterRemovingList = await profile.getIdentityAccessList();
    expect(afterRemovingList).to.not.include(identities[1]);
```